pyinjective/client/model/pagination.py

```diff
@@ -30,8 +31,9 @@ def __init__(
     def create_pagination_request(self) -> pagination_pb.PageRequest:
         page_request = pagination_pb.PageRequest()
 
-        if self.key is not None:
-            page_request.key = self.key.encode()
+        if self.encoded_page_key is not None and self.encoded_page_key != "":
+            page_key = base64.b64decode(self.encoded_page_key)
+            page_request.key = page_key
```

Comment:
This is a fair point, but wonder how could this happen. Think it's better to crash if some of these exceptions are risen

Reply:
I don't think this is a fair point. If there was an error in the base64 encoded string, the error would have happened before, when the response was converted into its JSON representation. Since we reached this point, we are sure the encoded pagination key can be decoded without issues.